### DISCUSSION TITLE
my functions not working anymore. Help.

### DISCUSSION THREAD

**Discussion by SupremmeNig** [psvks/NotifyControl#12] [General]
My functions not working anymore. INSTEAD of creating a function it does nothing.

Please help with this

**Comment by OrEye** [comment]
Hey there @SupremmeNig. We pushed recently an update that chaged almost every function.  
you might check the wiki / docs for more info. if the problem resists reply back.

WIKI [here](https://github.com/Obstronomic/NotifyControl/wiki)

Thanks.

**Comment by SupremmeNig** [comment]
tysm, problem solved
